content/docs/installation/supported-releases.md

```diff
@@ -17,25 +17,26 @@ cert-manager expects that ServerSideApply is enabled in the cluster for all vers
 
 <h2 id="supported-releases">Currently supported releases</h2>
 
-| Release  | Release Date |   End of Life   | [Supported Kubernetes versions][s] | [Supported OpenShift versions][s] |
-|----------|:------------:|:---------------:|:----------------------------------:|:---------------------------------:|
-| [1.11][] | Jan 11, 2023 | Release of 1.13 |            1.21 → 1.26             |            4.8 → 4.13             |
-| [1.10][] | Oct 17, 2022 | Release of 1.12 |            1.20 → 1.26             |            4.7 → 4.13             |
+| Release  | Release Date |      End of Life       | [Supported Kubernetes versions][s] | [Supported OpenShift versions][s] |
+|----------|:------------:|:----------------------:|:----------------------------------:|:---------------------------------:|
+| [1.12][] | May 19, 2024 | End of September, 2024 |            1.22 → 1.27             |            4.9 → 4.14             |
+| [1.11][] | Jan 11, 2023 |    Release of 1.13     |            1.21 → 1.26             |            4.8 → 4.13             |
```

Comment:
I think we say that we support kube 1.27 for release-1.11. I added tests for 1.27, but must have forgotten to update it here

Another nit: since we removed the 1.10 block, we can also remove the note about SSA apply below

Reply:
Also the links seem broken in preview version https://deploy-preview-1227--cert-manager-website.netlify.app/docs/installation/supported-releases/#old-releases
Not sure if that's a preview thing?

Reply:
Oops I forgot to add that file to the JSON manifest. I also added a line to the release process.